Description: Remove RPATH from library

Author: Sudip Mukherjee <sudipm.mukherjee@gmail.com>
Forwarded: not-needed

---

diff --git a/CMakeLists.txt b/CMakeLists.txt
index bb7b2a1..76b8f4c 100644
--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -124,19 +124,6 @@ set(KS_ICON_FIN    KS_icon_fin.svg)
 set(KS_LOGO        KS_logo_symbol.svg)
 set(KS_LOGO_LABEL  KS_logo_horizontal.svg)
 
-set(CMAKE_INSTALL_RPATH "${_LIBDIR}" "$ORIGIN")
-set(CMAKE_INSTALL_RPATH_USE_LINK_PATH TRUE)
-
-if (CMAKE_BUILD_TYPE MATCHES Package)
-
-    set(CMAKE_BUILD_WITH_INSTALL_RPATH TRUE)
-
-else (CMAKE_BUILD_TYPE MATCHES Package)
-
-    set(CMAKE_BUILD_WITH_INSTALL_RPATH FALSE)
-
-endif (CMAKE_BUILD_TYPE MATCHES Package)
-
 include_directories(${KS_DIR}/src/
                     ${KS_DIR}/build/src/)
 
